webapp/channels/src/components/initial_loading_screen/initial_loading_screen_class.ts

```diff
@@ -26,10 +26,23 @@ export class InitialLoadingScreenClass {
     constructor() {
         this.loadingScreenElement = document.getElementById('initialPageLoadingScreen');
         this.loadingAnimationElement = document.getElementById('initialPageLoadingAnimation');
-
         this.initialLoadingScreenCSS = document.getElementById('initialLoadingScreenCSS') as HTMLLinkElement | null;
 
         this.handleAnimationEndEvent = this.handleAnimationEndEvent.bind(this);
+
+        this.init();
+    }
+
+    private init() {
+        if (isDesktopApp()) {
+            // Let Mattermost desktop handle the loading screen
+            this.destroy();
+            return;
+        }
+
+        this.addAnimationEndListener();
+
+        this.start();
     }
 
```

Comment:
@M-ZubairAhmed I'm not familiar with the loading functionality here, could you add some comments how moving this to the `init` and calling explicitly fixes the issue?

Reply:
i added a comment @marianunez